Assets/MixedRealityToolkit/_Core/Extensions/ProcessExtensions.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if UNITY_EDITOR || !UNITY_WSA
```

Comment:
This isn't really needed unless you also want to prevent use at runtime (which we don't)
Can we avoid #if tags unless we really need them.

Reply:
I don't understand.  We do need them. This should be availible for any platform except Unity. Right now the only implementation is in the Editor.

Reply:
> avoid #if tags unless we really need them.

I agree.

Reply:
Why wasn't some of these concerns brought of in the initial PR?